src/layers/XR3DLayer/xr-layer-vertex.glsl

```diff
@@ -23,7 +24,7 @@ flat out vec3 transformed_eye;
 void main() {
 
   // Step 1: Standard MVP transformation (+ the scale matrix) to place the positions on your 2D screen ready for rasterization + fragment processing.
-  gl_Position = proj * view * model * scale * vec4(positions, 1.0);
+  gl_Position = proj * view * model * scale * resolution * vec4(positions, 1.0);
```

Comment:
The reason for splitting this is that `scale` encodes the pixel-based scaling of the volume while resolution is for "scaling up" to the volume's full resolution dimensions.  We can't make one matrix unfortunately because `scale` is used on the fragment shader (and pre-multiplying `resolution` would make that problematic).